apps/openmw/mwsound/openal_output.hpp

```diff
@@ -14,12 +14,13 @@
 
 namespace MWSound
 {
-    class SoundManager;
     class Sound;
     class Stream;
 
     class OpenAL_Output : public Sound_Output
     {
+        const DecoderProvider* mDecoderProvider;
```

Comment:
More a stylistic thing that doesn't really matter too much, but this can be a reference instead of a pointer. It would avoid accidental changes, and make it clear that it's set once on initialization and never changed.